taipy/core/exceptions/exceptions.py

```diff
@@ -373,6 +373,47 @@ class FileCannotBeRead(Exception):
     """Raised when a file cannot be read."""
 
 
+class ExportFolderAlreadyExists(Exception):
+    """Raised when the export folder already exists."""
+
+    def __init__(self, folder_path: str, scenario_id: str):
+        self.message = (
+            f"Folder '{folder_path}' already exists and can not be used to export scenario '{scenario_id}'."
+            " Please use the 'override' parameter to override it."
+        )
+
+
+class EntitiesToBeImportAlredyExist(Exception):
+    """Raised when entities in the scenario to be imported have already exists"""
+
+    def __init__(self, folder_path):
+        self.message = f"The import folder {folder_path} contains entities that have already existed."
+
+
+class DataToBeImportAlredyExist(Exception):
+    """Raised when data files in the scenario to be imported have already exists"""
+
+    def __init__(self, folder_path):
+        self.message = (
+            f"The import folder {folder_path} contains data files that have already existed."
+            " Please use the 'override' parameter to override those."
+        )
+
+
+class ImportFolderDoesntContainAnyScenario(Exception):
+    """Raised when the import folder doesn't contain any scenario"""
+
+    def __init__(self, folder_path):
+        self.message = f"The import folder {folder_path} doesn't contain any scenario."
+
+
+class ImportScenarioDoesntHaveAVersion(Exception):
+    """Raised when the import folder doesn't contain any scenario"""
+
+    def __init__(self, folder_path):
```

Comment:
Please take the opportunity of this PR to add the return type for `__init__` methods. Thx

Reply:
We are not having the `-> None` on the Exception classes. Should we do it all?

Reply:
Yes, I think so. 
I would suggest doing this kind of changes iteratively, small by small, instead of doing everything in the same big PR. That's why I was talking about taking the opportunity.
However, since your PR is already big enough if you believe it will make it too difficult to review, we can decide not to do it in this PR.

Nothing is blocking for me anyway. It's up to you.